ckanext/datapackager/tests/controllers/test_datapackage.py

```diff
@@ -61,7 +61,8 @@ def test_download_datapackage(self, app):
 
         # Open and validate the response as a JSON.
         dp = datapackage.DataPackage(json.loads(response.body))
-        dp.validate()
+        # TODO: Check why the datapackage is not valid according to datapackage module
+        #dp.validate()
```

Comment:
This sounds important. What validation errors are being raised here?

Reply:
I checked and the problem was that the factories.Dataset was being created without the maintainer information and the license information. This generated a datapackage with a empty list of contributor as the license had None in the name and url. I fixed and put but the validation step.